api/TaskTracker/TaskTracker.Data/Entities/UserEntity.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace TaskTracker.Data.Entities;
+public class UserEntity : IdentityUser<int>
+{
+    [Required, StringLength(100)]
+    public string FirstName { get; set; } = string.Empty;
```

Comment:
remove string.Empty assignment, since you are adding a Required attribute and don't want for compiler to show warning, just make it required on property level:
[StringLength(100)]
public required string FirstName { get; set; }

The same logic for other similar fields